File: HomeWorkout/lib/models/ExerciseModel.dart
import 'package:flutter/cupertino.dart';

class ExerciseModel {
  int id;
  String name;
  String tag;
  int numberOfTimes;
  bool isChoosen;
  int seconds;
  String imageSource;
  ExerciseModel(
      {this.id,
      this.name,
      this.tag,
      this.isChoosen,
      this.numberOfTimes,
      this.seconds,
      this.imageSource});
  ExerciseModel.manual(this.id, this.name, this.tag);

  factory ExerciseModel.fromJson(Map<String, dynamic> json) {
    return ExerciseModel(
        id: json['id'] as int,
        tag: json['tag'] as String,
        name: json['name'] as String,
        seconds: json['seconds'] as int,
        isChoosen: json['isChoosen'] as bool,
        numberOfTimes: json['isChoosen'] as int,
        imageSource: json['imageSource'] as String);
  }
}

File: HomeWorkoutBackend/HomeWorkoutBackend/Models/ExerciseModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace HomeWorkOutApi.NetCore.Models
{
    public class ExerciseModel
    {
        public int Id { get; set; }
        public DateTime CreationTime {get; set; }
        public string Name { get; set; }
        public string Tag { get; set; }
        public string Description { get; set; }
        public int NumberOfTimes { get; set; }
        public bool IsChoosen { get; set; }
        public int Seconds { get; set; }
        public string ImageSource { get; set; }
        public ExerciseModel()
        {

        }
    }
}